gtfs-realtime/spec/en/trip-modifications.md

```diff
@@ -0,0 +1,64 @@
+# Trip Modifications
+
+A `TripModifications` message identifies a list of similar `trip_ids` from the (CSV) GTFS sharing the same stop pattern, which are all affected by a particular modification, such as a detour.
+
+## SLO: Service-level objective
+
+The frequency of data updates is expected to be approximately hourly (~24 times/day). Ingestion time may depend on the total number of affected trips. Consumers are expected to ingest a single TripModification within 5 minutes, and a feed with hundreds of detours within 20 minutes.
```

Comment:
question: not a copy edit, but is the expectation that this data does not live within a `TripUpdates` feed which updates more frequently, or only that the data itself is not changing even if the feed itself is?

Reply:
We're not changing anything on the spec in regard on how feeds can contain multiple types of entities or not : ["A feed may, although not required to, combine entities of different types."](https://github.com/google/transit/pull/403/files#diff-90f282d10f519d763309d02a8161020994bcf89cfc6d45dabc6376e2b0c5492dR22) Swiftly merges TripModif with TripUpdates but we support any configuration. And yes, TripUpdates should update as frequently as needed and most likely will update more frequently than TripModifications. 

This section is to "manage expectations" that consumer can't ingest that type of data in seconds like we do for all other types of RT feeds. It's inspired by [Service-Changes](https://docs.google.com/document/d/1oPfQ6Xvui0g3xiy1pNiyu5cWFLhvrTVyvYsMVKGyGWM/edit#heading=h.dk9syw20pc6j), but I'm not sure how required it is.